index.html

```diff
@@ -26,7 +26,26 @@
       <li>Capitol Hill</li>
       <li>Alki</li>
     </ul>
-    <p id="inspiration">Remember! Life might get hard, but always swim upstream!</p>
-    <script src="app.js" type="text/javascript"></script>
+    <div class="form">
```

Comment:
I would avoid naming classes after element tags, especially if you're adding that class onto a different kind of element.